security/saml2aws/Portfile

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/Versent/saml2aws 2.30.0 v
+revision            0
+categories          security
+platforms           darwin
+supported_archs     arm64 x86_64
```

Comment:
You can skip `supported_archs` - the `golang` portgroup should do the right thing per architecture.

Reply:
Thanks, I've removed this.